build.gradle.kts

```diff
@@ -23,9 +23,35 @@
 group = "dev.hypera"
 version = "0.2.0-SNAPSHOT"
 
+//Minestom
```

Comment:
These should be in a version catalogue, not as variables in this file.

Reply:
Could you give a example?

Reply:
An example can be found in our [Chameleon](https://github.com/ChameleonFramework/Chameleon/blob/54fb4236e17595132b387886d89c3bbfd65222f7/gradle/libs.versions.toml) library.

Reply:
> An example can be found in our [Chameleon](https://github.com/ChameleonFramework/Chameleon/blob/54fb4236e17595132b387886d89c3bbfd65222f7/gradle/libs.versions.toml) library.

Thanks for the example, i tried my best but it did not work for me. Maybe you could implement this? I saw you did the chameloen project too, so you know what to do xD